postgis/gserialized_gist_nd.c

```diff
@@ -1365,6 +1367,56 @@ Datum gserialized_gist_distance(PG_FUNCTION_ARGS)
 	PG_RETURN_FLOAT8(distance);
 }
 
+PG_FUNCTION_INFO_V1(gserialized_gist_sortsupport);
+static int
+hash_cmp(Datum a, Datum b, SortSupport ssup)
+{
+  if (a > b)
+    return 1;
+  else if (a < b)
+    return -1;
+  else
+    return 0;
+}
+
+static bool
+hash_abbrev_abort(int memtupcount, SortSupport SortSupport)
+{
+  return false;
+}
+
+static Datum
+hash_abbrev_convert(Datum original, SortSupport ssup)
+{
+	return (Datum)0;
```

Comment:
GIDX is used to store GIST Nd so it looks like you can consider using `gbox_get_sortable_hash` for now
https://github.com/postgis/postgis/blob/45e491242d9d63beb7eac0961b16439d00c93bd4/liblwgeom/gbox.c#L893